metricbeat/docs/modules/ceph.asciidoc

```diff
@@ -0,0 +1,57 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[metricbeat-module-ceph]]
+== ceph Module
+
+This is the ceph Module.
+
+We used code/reference from:
```

Comment:
Since I'll change to use Rest API, this reference will be no longer necessary indeed. I'll remove.